public/content/exjobbs/2023_broman_astrocasting.mdx

```diff
@@ -6,7 +6,7 @@ period: "Spring 2024"
 number_of_students: "2"
 contact: [ "emmbr26", "alebo68" ]
 
-finished: false
+finished: true

```

Comment:
This is not really finished though.  Did we have an option to hide it instead?

Reply:
In case we might want to offer it at a later time

Reply:
We don't have it yet, but I'll add